config/templates/needsReviewWarning.md

```diff
@@ -1,3 +1 @@
-Hello @{assignees}, @{author} updated this pull request over {days} days ago in response to your review, but you haven't checked back in on their changes! Please leave some feedback for them or mark this pull request as ready for merging.
-
-Thank you for reviewing this pull request!
+Hello @{assignees}, @{author} updated this pull request over {days} days ago in response to your review, but you haven't checked back in on their changes! Please leave some feedback for them or mark this pull request as ready for merging. Thanks!
```

Comment:
"ready for merging" feels off to me, maybe just "Please leave some feedback on next steps for this pull request"?

Reply:
Sure.

Reply:
WFM